pkg/build/rpm.go

```diff
@@ -44,15 +67,38 @@ func (b *Builder) copyRPMs() error {
 		return fmt.Errorf("getting rpm file names: %w", err)
 	}
 
-	for _, rpm := range rpmFileNames {
-		sourcePath := filepath.Join(rpmSourceDir, rpm)
-		destPath := filepath.Join(rpmDestDir, rpm)
+	err = b.copyRPMs(rpmSourceDir, rpmDestDir, rpmFileNames)
+	if err != nil {
+		return fmt.Errorf("copying RPMs over: %w", err)
+	}
 
-		err = copyFile(sourcePath, destPath)
-		if err != nil {
-			return fmt.Errorf("copying file %s: %w", sourcePath, err)
-		}
+	err = b.writeRPMScript(rpmFileNames)
+	if err != nil {
+		return fmt.Errorf("writing the rpm install script: %w", err)
 	}
 
 	return nil
 }
+
+func (b *Builder) writeRPMScript(rpmFileNamesArray []string) error {
+	rpmFileNamesString := strings.Join(rpmFileNamesArray, " ")
+	values := struct {
+		RPMs string
+	}{
+		RPMs: rpmFileNamesString,
+	}
+
+	writtenFilename, err := b.writeCombustionFile(modifyRPMScriptName, modifyRPMScript, &values)
+	if err != nil {
+		return fmt.Errorf("writing rpm script %s: %w", modifyRPMScriptName, err)
+	}
+	err = os.Chmod(writtenFilename, modifyScriptMode)
+	if err != nil {
+		return fmt.Errorf("changing permissions on the rpm script %s: %w", modifyRPMScriptName, err)
+	}
+
+	fmt.Println(writtenFilename, "xaksdokasd")
```

Comment:
I'm going to assume this wasn't meant to be in the PR :)

Reply:
LMAO, my mistake on that, knew I had to have missed something